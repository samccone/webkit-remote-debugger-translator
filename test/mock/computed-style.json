{"result":{"computedStyle":[{"name":"alt","value":"''"},{"name":"animation-delay","value":"0s"},{"name":"animation-direction","value":"normal"},{"name":"animation-duration","value":"0s"},{"name":"animation-fill-mode","value":"none"},{"name":"animation-iteration-count","value":"1"},{"name":"animation-name","value":"none"},{"name":"animation-play-state","value":"running"},{"name":"animation-timing-function","value":"ease"},{"name":"background-attachment","value":"scroll"},{"name":"background-blend-mode","value":"normal"},{"name":"background-clip","value":"border-box"},{"name":"background-color","value":"rgba(0, 0, 0, 0)"},{"name":"background-image","value":"none"},{"name":"background-origin","value":"padding-box"},{"name":"background-position","value":"0% 0%"},{"name":"background-repeat","value":"repeat"},{"name":"background-size","value":"auto"},{"name":"border-bottom-color","value":"rgb(0, 0, 0)"},{"name":"border-bottom-left-radius","value":"0px"},{"name":"border-bottom-right-radius","value":"0px"},{"name":"border-bottom-style","value":"none"},{"name":"border-bottom-width","value":"0px"},{"name":"border-collapse","value":"separate"},{"name":"border-image-outset","value":"0px"},{"name":"border-image-repeat","value":"stretch"},{"name":"border-image-slice","value":"100%"},{"name":"border-image-source","value":"none"},{"name":"border-image-width","value":"1"},{"name":"border-left-color","value":"rgb(0, 0, 0)"},{"name":"border-left-style","value":"none"},{"name":"border-left-width","value":"0px"},{"name":"border-right-color","value":"rgb(0, 0, 0)"},{"name":"border-right-style","value":"none"},{"name":"border-right-width","value":"0px"},{"name":"border-top-color","value":"rgb(0, 0, 0)"},{"name":"border-top-left-radius","value":"0px"},{"name":"border-top-right-radius","value":"0px"},{"name":"border-top-style","value":"none"},{"name":"border-top-width","value":"0px"},{"name":"bottom","value":"auto"},{"name":"box-shadow","value":"none"},{"name":"box-sizing","value":"content-box"},{"name":"caption-side","value":"top"},{"name":"clear","value":"none"},{"name":"clip","value":"auto"},{"name":"color","value":"rgb(0, 0, 0)"},{"name":"cursor","value":"auto"},{"name":"direction","value":"ltr"},{"name":"display","value":"inline"},{"name":"empty-cells","value":"show"},{"name":"float","value":"none"},{"name":"font-family","value":"Roboto, sans-serif"},{"name":"font-size","value":"22px"},{"name":"font-style","value":"normal"},{"name":"font-synthesis","value":"style weight"},{"name":"font-variant","value":"normal"},{"name":"font-weight","value":"normal"},{"name":"height","value":"auto"},{"name":"image-rendering","value":"auto"},{"name":"left","value":"auto"},{"name":"letter-spacing","value":"-0.2199999988079071px"},{"name":"line-height","value":"25px"},{"name":"list-style-image","value":"none"},{"name":"list-style-position","value":"outside"},{"name":"list-style-type","value":"disc"},{"name":"margin-bottom","value":"0px"},{"name":"margin-left","value":"0px"},{"name":"margin-right","value":"0px"},{"name":"margin-top","value":"0px"},{"name":"max-height","value":"none"},{"name":"max-width","value":"none"},{"name":"min-height","value":"0px"},{"name":"min-width","value":"0px"},{"name":"opacity","value":"1"},{"name":"orphans","value":"auto"},{"name":"outline-color","value":"rgb(0, 0, 0)"},{"name":"outline-offset","value":"0px"},{"name":"outline-style","value":"none"},{"name":"outline-width","value":"0px"},{"name":"overflow-wrap","value":"normal"},{"name":"overflow-x","value":"visible"},{"name":"overflow-y","value":"visible"},{"name":"padding-bottom","value":"0px"},{"name":"padding-left","value":"0px"},{"name":"padding-right","value":"0px"},{"name":"padding-top","value":"0px"},{"name":"page-break-after","value":"auto"},{"name":"page-break-before","value":"auto"},{"name":"page-break-inside","value":"auto"},{"name":"pointer-events","value":"auto"},{"name":"position","value":"static"},{"name":"resize","value":"none"},{"name":"right","value":"auto"},{"name":"speak","value":"normal"},{"name":"table-layout","value":"auto"},{"name":"tab-size","value":"8"},{"name":"text-align","value":"start"},{"name":"text-decoration","value":"none"},{"name":"-webkit-text-decoration-line","value":"none"},{"name":"-webkit-text-decoration-style","value":"solid"},{"name":"-webkit-text-decoration-color","value":"rgb(0, 0, 0)"},{"name":"-webkit-text-decoration-skip","value":"auto"},{"name":"-webkit-text-underline-position","value":"auto"},{"name":"text-indent","value":"0px"},{"name":"text-rendering","value":"auto"},{"name":"text-shadow","value":"none"},{"name":"text-overflow","value":"clip"},{"name":"text-transform","value":"none"},{"name":"top","value":"auto"},{"name":"transform","value":"none"},{"name":"transform-origin","value":"0px 0px"},{"name":"transform-style","value":"flat"},{"name":"transition-delay","value":"0s"},{"name":"transition-duration","value":"0s"},{"name":"transition-property","value":"all"},{"name":"transition-timing-function","value":"ease"},{"name":"unicode-bidi","value":"normal"},{"name":"vertical-align","value":"baseline"},{"name":"visibility","value":"visible"},{"name":"white-space","value":"normal"},{"name":"widows","value":"auto"},{"name":"width","value":"auto"},{"name":"word-break","value":"normal"},{"name":"word-spacing","value":"0px"},{"name":"word-wrap","value":"normal"},{"name":"-webkit-scroll-snap-type","value":"none"},{"name":"-webkit-scroll-snap-points-x","value":"none"},{"name":"-webkit-scroll-snap-points-y","value":"none"},{"name":"-webkit-scroll-snap-destination","value":"0px 0px"},{"name":"-webkit-scroll-snap-coordinate","value":"none"},{"name":"z-index","value":"auto"},{"name":"zoom","value":"1"},{"name":"-webkit-animation-delay","value":"0s"},{"name":"-webkit-animation-direction","value":"normal"},{"name":"-webkit-animation-duration","value":"0s"},{"name":"-webkit-animation-fill-mode","value":"none"},{"name":"-webkit-animation-iteration-count","value":"1"},{"name":"-webkit-animation-name","value":"none"},{"name":"-webkit-animation-play-state","value":"running"},{"name":"-webkit-animation-timing-function","value":"ease"},{"name":"-webkit-appearance","value":"none"},{"name":"-webkit-backface-visibility","value":"visible"},{"name":"-webkit-background-clip","value":"border-box"},{"name":"-webkit-background-composite","value":"source-over"},{"name":"-webkit-background-origin","value":"padding-box"},{"name":"-webkit-background-size","value":"auto"},{"name":"mix-blend-mode","value":"normal"},{"name":"isolation","value":"auto"},{"name":"-webkit-border-fit","value":"border"},{"name":"-webkit-border-horizontal-spacing","value":"0px"},{"name":"-webkit-border-image","value":"none"},{"name":"-webkit-border-vertical-spacing","value":"0px"},{"name":"-webkit-box-align","value":"stretch"},{"name":"-webkit-box-decoration-break","value":"slice"},{"name":"-webkit-box-direction","value":"normal"},{"name":"-webkit-box-flex","value":"0"},{"name":"-webkit-box-flex-group","value":"1"},{"name":"-webkit-box-lines","value":"single"},{"name":"-webkit-box-ordinal-group","value":"1"},{"name":"-webkit-box-orient","value":"horizontal"},{"name":"-webkit-box-pack","value":"start"},{"name":"-webkit-box-reflect","value":"none"},{"name":"-webkit-box-shadow","value":"none"},{"name":"-webkit-clip-path","value":"none"},{"name":"-webkit-color-correction","value":"default"},{"name":"-webkit-column-break-after","value":"auto"},{"name":"-webkit-column-break-before","value":"auto"},{"name":"-webkit-column-break-inside","value":"auto"},{"name":"-webkit-column-axis","value":"auto"},{"name":"column-count","value":"auto"},{"name":"column-fill","value":"balance"},{"name":"column-gap","value":"normal"},{"name":"column-progression","value":"normal"},{"name":"column-rule-color","value":"rgb(0, 0, 0)"},{"name":"column-rule-style","value":"none"},{"name":"column-rule-width","value":"0px"},{"name":"column-span","value":"none"},{"name":"column-width","value":"auto"},{"name":"-webkit-cursor-visibility","value":"auto"},{"name":"align-content","value":"start"},{"name":"align-items","value":"start"},{"name":"align-self","value":"start"},{"name":"flex-basis","value":"auto"},{"name":"flex-grow","value":"0"},{"name":"flex-shrink","value":"1"},{"name":"flex-direction","value":"row"},{"name":"flex-wrap","value":"nowrap"},{"name":"justify-content","value":"start"},{"name":"justify-self","value":"start"},{"name":"justify-items","value":"start"},{"name":"-webkit-filter","value":"none"},{"name":"-webkit-backdrop-filter","value":"none"},{"name":"-webkit-font-kerning","value":"auto"},{"name":"-webkit-font-smoothing","value":"antialiased"},{"name":"-webkit-font-variant-ligatures","value":"normal"},{"name":"-webkit-hyphenate-character","value":"auto"},{"name":"-webkit-hyphenate-limit-after","value":"auto"},{"name":"-webkit-hyphenate-limit-before","value":"auto"},{"name":"-webkit-hyphenate-limit-lines","value":"no-limit"},{"name":"-webkit-hyphens","value":"manual"},{"name":"-webkit-initial-letter","value":"normal"},{"name":"-webkit-line-align","value":"none"},{"name":"-webkit-line-box-contain","value":"block inline replaced"},{"name":"-webkit-line-break","value":"auto"},{"name":"-webkit-line-clamp","value":"none"},{"name":"-webkit-line-grid","value":"none"},{"name":"-webkit-line-snap","value":"none"},{"name":"-webkit-locale","value":"en"},{"name":"-webkit-margin-before-collapse","value":"collapse"},{"name":"-webkit-margin-after-collapse","value":"collapse"},{"name":"-webkit-marquee-direction","value":"auto"},{"name":"-webkit-marquee-increment","value":"6px"},{"name":"-webkit-marquee-repetition","value":"infinite"},{"name":"-webkit-marquee-style","value":"scroll"},{"name":"-webkit-mask-box-image","value":"none"},{"name":"-webkit-mask-box-image-outset","value":"0px"},{"name":"-webkit-mask-box-image-repeat","value":"stretch"},{"name":"-webkit-mask-box-image-slice","value":"0 fill"},{"name":"-webkit-mask-box-image-source","value":"none"},{"name":"-webkit-mask-box-image-width","value":"auto"},{"name":"-webkit-mask-clip","value":"border-box"},{"name":"-webkit-mask-composite","value":"source-over"},{"name":"-webkit-mask-image","value":"none"},{"name":"-webkit-mask-origin","value":"border-box"},{"name":"-webkit-mask-position","value":"0% 0%"},{"name":"-webkit-mask-repeat","value":"repeat"},{"name":"-webkit-mask-size","value":"auto"},{"name":"-webkit-mask-source-type","value":"alpha"},{"name":"-webkit-nbsp-mode","value":"normal"},{"name":"order","value":"0"},{"name":"-webkit-overflow-scrolling","value":"auto"},{"name":"perspective","value":"none"},{"name":"perspective-origin","value":"0px 0px"},{"name":"-webkit-print-color-adjust","value":"economy"},{"name":"-webkit-rtl-ordering","value":"logical"},{"name":"-webkit-touch-callout","value":"default"},{"name":"-webkit-shape-outside","value":"none"},{"name":"-webkit-tap-highlight-color","value":"rgba(0, 0, 0, 0)"},{"name":"-webkit-text-combine","value":"none"},{"name":"-webkit-text-decorations-in-effect","value":"none"},{"name":"-webkit-text-emphasis-color","value":"rgb(0, 0, 0)"},{"name":"-webkit-text-emphasis-position","value":"over right"},{"name":"-webkit-text-emphasis-style","value":"none"},{"name":"-webkit-text-fill-color","value":"rgb(0, 0, 0)"},{"name":"-webkit-text-orientation","value":"vertical-right"},{"name":"-webkit-text-security","value":"none"},{"name":"-webkit-text-size-adjust","value":"auto"},{"name":"-webkit-text-stroke-color","value":"rgb(0, 0, 0)"},{"name":"-webkit-text-stroke-width","value":"0px"},{"name":"-webkit-transform-style","value":"flat"},{"name":"-webkit-transition-delay","value":"0s"},{"name":"-webkit-transition-duration","value":"0s"},{"name":"-webkit-transition-property","value":"all"},{"name":"-webkit-transition-timing-function","value":"ease"},{"name":"-webkit-user-drag","value":"auto"},{"name":"-webkit-user-modify","value":"read-only"},{"name":"-webkit-user-select","value":"text"},{"name":"-webkit-writing-mode","value":"horizontal-tb"},{"name":"-webkit-flow-into","value":"none"},{"name":"-webkit-flow-from","value":"none"},{"name":"-webkit-region-break-after","value":"auto"},{"name":"-webkit-region-break-before","value":"auto"},{"name":"-webkit-region-break-inside","value":"auto"},{"name":"-webkit-region-fragment","value":"auto"},{"name":"-webkit-shape-margin","value":"0px"},{"name":"-webkit-shape-image-threshold","value":"0"},{"name":"buffered-rendering","value":"auto"},{"name":"clip-path","value":"none"},{"name":"clip-rule","value":"nonzero"},{"name":"cx","value":"0px"},{"name":"cy","value":"0px"},{"name":"mask","value":"none"},{"name":"filter","value":"none"},{"name":"flood-color","value":"rgb(0, 0, 0)"},{"name":"flood-opacity","value":"1"},{"name":"lighting-color","value":"rgb(255, 255, 255)"},{"name":"stop-color","value":"rgb(0, 0, 0)"},{"name":"stop-opacity","value":"1"},{"name":"color-interpolation","value":"srgb"},{"name":"color-interpolation-filters","value":"linearrgb"},{"name":"color-rendering","value":"auto"},{"name":"fill","value":"#000000"},{"name":"fill-opacity","value":"1"},{"name":"fill-rule","value":"nonzero"},{"name":"marker-end","value":"none"},{"name":"marker-mid","value":"none"},{"name":"marker-start","value":"none"},{"name":"mask-type","value":"luminance"},{"name":"paint-order","value":"normal"},{"name":"r","value":"0px"},{"name":"rx","value":"0px"},{"name":"ry","value":"0px"},{"name":"shape-rendering","value":"auto"},{"name":"stroke","value":"none"},{"name":"stroke-dasharray","value":"none"},{"name":"stroke-dashoffset","value":"0px"},{"name":"stroke-linecap","value":"butt"},{"name":"stroke-linejoin","value":"miter"},{"name":"stroke-miterlimit","value":"4"},{"name":"stroke-opacity","value":"1"},{"name":"stroke-width","value":"1px"},{"name":"alignment-baseline","value":"auto"},{"name":"baseline-shift","value":"baseline"},{"name":"dominant-baseline","value":"auto"},{"name":"kerning","value":"0"},{"name":"text-anchor","value":"start"},{"name":"writing-mode","value":"lr-tb"},{"name":"glyph-orientation-horizontal","value":"0deg"},{"name":"glyph-orientation-vertical","value":"auto"},{"name":"-webkit-svg-shadow","value":"none"},{"name":"vector-effect","value":"none"},{"name":"x","value":"0px"},{"name":"y","value":"0px"}]},"id":73}	
