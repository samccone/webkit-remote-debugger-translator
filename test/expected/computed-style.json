{
    "id": 40,
    "result": {
        "computedStyle": [
            {
                "name": "animation-delay",
                "value": "0s"
            },
            {
                "name": "animation-direction",
                "value": "normal"
            },
            {
                "name": "animation-duration",
                "value": "0s"
            },
            {
                "name": "animation-fill-mode",
                "value": "none"
            },
            {
                "name": "animation-iteration-count",
                "value": "1"
            },
            {
                "name": "animation-name",
                "value": "none"
            },
            {
                "name": "animation-play-state",
                "value": "running"
            },
            {
                "name": "animation-timing-function",
                "value": "ease"
            },
            {
                "name": "background-attachment",
                "value": "scroll"
            },
            {
                "name": "background-blend-mode",
                "value": "normal"
            },
            {
                "name": "background-clip",
                "value": "border-box"
            },
            {
                "name": "background-color",
                "value": "rgb(255, 0, 0)"
            },
            {
                "name": "background-image",
                "value": "none"
            },
            {
                "name": "background-origin",
                "value": "padding-box"
            },
            {
                "name": "background-position",
                "value": "0% 0%"
            },
            {
                "name": "background-repeat",
                "value": "repeat"
            },
            {
                "name": "background-size",
                "value": "auto"
            },
            {
                "name": "border-bottom-color",
                "value": "rgb(128, 0, 128)"
            },
            {
                "name": "border-bottom-left-radius",
                "value": "0px"
            },
            {
                "name": "border-bottom-right-radius",
                "value": "0px"
            },
            {
                "name": "border-bottom-style",
                "value": "solid"
            },
            {
                "name": "border-bottom-width",
                "value": "2px"
            },
            {
                "name": "border-collapse",
                "value": "separate"
            },
            {
                "name": "border-image-outset",
                "value": "0px"
            },
            {
                "name": "border-image-repeat",
                "value": "stretch"
            },
            {
                "name": "border-image-slice",
                "value": "100%"
            },
            {
                "name": "border-image-source",
                "value": "none"
            },
            {
                "name": "border-image-width",
                "value": "1"
            },
            {
                "name": "border-left-color",
                "value": "rgb(128, 0, 128)"
            },
            {
                "name": "border-left-style",
                "value": "solid"
            },
            {
                "name": "border-left-width",
                "value": "2px"
            },
            {
                "name": "border-right-color",
                "value": "rgb(128, 0, 128)"
            },
            {
                "name": "border-right-style",
                "value": "solid"
            },
            {
                "name": "border-right-width",
                "value": "2px"
            },
            {
                "name": "border-top-color",
                "value": "rgb(128, 0, 128)"
            },
            {
                "name": "border-top-left-radius",
                "value": "0px"
            },
            {
                "name": "border-top-right-radius",
                "value": "0px"
            },
            {
                "name": "border-top-style",
                "value": "solid"
            },
            {
                "name": "border-top-width",
                "value": "2px"
            },
            {
                "name": "bottom",
                "value": "auto"
            },
            {
                "name": "box-shadow",
                "value": "none"
            },
            {
                "name": "box-sizing",
                "value": "content-box"
            },
            {
                "name": "caption-side",
                "value": "top"
            },
            {
                "name": "clear",
                "value": "none"
            },
            {
                "name": "clip",
                "value": "auto"
            },
            {
                "name": "color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "cursor",
                "value": "auto"
            },
            {
                "name": "direction",
                "value": "ltr"
            },
            {
                "name": "display",
                "value": "block"
            },
            {
                "name": "empty-cells",
                "value": "show"
            },
            {
                "name": "float",
                "value": "none"
            },
            {
                "name": "font-family",
                "value": "Times"
            },
            {
                "name": "font-kerning",
                "value": "auto"
            },
            {
                "name": "font-size",
                "value": "10px"
            },
            {
                "name": "font-stretch",
                "value": "normal"
            },
            {
                "name": "font-style",
                "value": "normal"
            },
            {
                "name": "font-variant",
                "value": "normal"
            },
            {
                "name": "font-variant-ligatures",
                "value": "normal"
            },
            {
                "name": "font-weight",
                "value": "bold"
            },
            {
                "name": "height",
                "value": "13px"
            },
            {
                "name": "image-rendering",
                "value": "auto"
            },
            {
                "name": "isolation",
                "value": "auto"
            },
            {
                "name": "left",
                "value": "auto"
            },
            {
                "name": "letter-spacing",
                "value": "normal"
            },
            {
                "name": "line-height",
                "value": "normal"
            },
            {
                "name": "list-style-image",
                "value": "none"
            },
            {
                "name": "list-style-position",
                "value": "outside"
            },
            {
                "name": "list-style-type",
                "value": "disc"
            },
            {
                "name": "margin-bottom",
                "value": "6.7px"
            },
            {
                "name": "margin-left",
                "value": "0px"
            },
            {
                "name": "margin-right",
                "value": "0px"
            },
            {
                "name": "margin-top",
                "value": "6.7px"
            },
            {
                "name": "max-height",
                "value": "none"
            },
            {
                "name": "max-width",
                "value": "none"
            },
            {
                "name": "min-height",
                "value": "0px"
            },
            {
                "name": "min-width",
                "value": "0px"
            },
            {
                "name": "mix-blend-mode",
                "value": "normal"
            },
            {
                "name": "motion-offset",
                "value": "0px"
            },
            {
                "name": "motion-path",
                "value": "none"
            },
            {
                "name": "motion-rotation",
                "value": "auto 0deg"
            },
            {
                "name": "object-fit",
                "value": "fill"
            },
            {
                "name": "object-position",
                "value": "50% 50%"
            },
            {
                "name": "opacity",
                "value": "1"
            },
            {
                "name": "orphans",
                "value": "auto"
            },
            {
                "name": "outline-color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "outline-offset",
                "value": "0px"
            },
            {
                "name": "outline-style",
                "value": "none"
            },
            {
                "name": "outline-width",
                "value": "0px"
            },
            {
                "name": "overflow-wrap",
                "value": "normal"
            },
            {
                "name": "overflow-x",
                "value": "visible"
            },
            {
                "name": "overflow-y",
                "value": "visible"
            },
            {
                "name": "padding-bottom",
                "value": "0px"
            },
            {
                "name": "padding-left",
                "value": "0px"
            },
            {
                "name": "padding-right",
                "value": "0px"
            },
            {
                "name": "padding-top",
                "value": "0px"
            },
            {
                "name": "page-break-after",
                "value": "auto"
            },
            {
                "name": "page-break-before",
                "value": "auto"
            },
            {
                "name": "page-break-inside",
                "value": "auto"
            },
            {
                "name": "pointer-events",
                "value": "auto"
            },
            {
                "name": "position",
                "value": "static"
            },
            {
                "name": "resize",
                "value": "none"
            },
            {
                "name": "right",
                "value": "auto"
            },
            {
                "name": "speak",
                "value": "normal"
            },
            {
                "name": "table-layout",
                "value": "auto"
            },
            {
                "name": "tab-size",
                "value": "8"
            },
            {
                "name": "text-align",
                "value": "start"
            },
            {
                "name": "text-decoration",
                "value": "none"
            },
            {
                "name": "text-indent",
                "value": "0px"
            },
            {
                "name": "text-rendering",
                "value": "auto"
            },
            {
                "name": "text-shadow",
                "value": "none"
            },
            {
                "name": "text-overflow",
                "value": "clip"
            },
            {
                "name": "text-transform",
                "value": "none"
            },
            {
                "name": "top",
                "value": "auto"
            },
            {
                "name": "touch-action",
                "value": "auto"
            },
            {
                "name": "transition-delay",
                "value": "0s"
            },
            {
                "name": "transition-duration",
                "value": "0s"
            },
            {
                "name": "transition-property",
                "value": "all"
            },
            {
                "name": "transition-timing-function",
                "value": "ease"
            },
            {
                "name": "unicode-bidi",
                "value": "normal"
            },
            {
                "name": "vertical-align",
                "value": "baseline"
            },
            {
                "name": "visibility",
                "value": "visible"
            },
            {
                "name": "white-space",
                "value": "normal"
            },
            {
                "name": "widows",
                "value": "1"
            },
            {
                "name": "width",
                "value": "1420px"
            },
            {
                "name": "will-change",
                "value": "auto"
            },
            {
                "name": "word-break",
                "value": "normal"
            },
            {
                "name": "word-spacing",
                "value": "0px"
            },
            {
                "name": "word-wrap",
                "value": "normal"
            },
            {
                "name": "z-index",
                "value": "auto"
            },
            {
                "name": "zoom",
                "value": "1"
            },
            {
                "name": "-webkit-appearance",
                "value": "none"
            },
            {
                "name": "backface-visibility",
                "value": "visible"
            },
            {
                "name": "-webkit-background-clip",
                "value": "border-box"
            },
            {
                "name": "-webkit-background-composite",
                "value": "source-over"
            },
            {
                "name": "-webkit-background-origin",
                "value": "padding-box"
            },
            {
                "name": "-webkit-border-horizontal-spacing",
                "value": "0px"
            },
            {
                "name": "-webkit-border-image",
                "value": "none"
            },
            {
                "name": "-webkit-border-vertical-spacing",
                "value": "0px"
            },
            {
                "name": "-webkit-box-align",
                "value": "stretch"
            },
            {
                "name": "-webkit-box-decoration-break",
                "value": "slice"
            },
            {
                "name": "-webkit-box-direction",
                "value": "normal"
            },
            {
                "name": "-webkit-box-flex",
                "value": "0"
            },
            {
                "name": "-webkit-box-flex-group",
                "value": "1"
            },
            {
                "name": "-webkit-box-lines",
                "value": "single"
            },
            {
                "name": "-webkit-box-ordinal-group",
                "value": "1"
            },
            {
                "name": "-webkit-box-orient",
                "value": "horizontal"
            },
            {
                "name": "-webkit-box-pack",
                "value": "start"
            },
            {
                "name": "-webkit-box-reflect",
                "value": "none"
            },
            {
                "name": "-webkit-clip-path",
                "value": "none"
            },
            {
                "name": "-webkit-column-break-after",
                "value": "auto"
            },
            {
                "name": "-webkit-column-break-before",
                "value": "auto"
            },
            {
                "name": "-webkit-column-break-inside",
                "value": "auto"
            },
            {
                "name": "-webkit-column-count",
                "value": "auto"
            },
            {
                "name": "-webkit-column-gap",
                "value": "normal"
            },
            {
                "name": "-webkit-column-rule-color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "-webkit-column-rule-style",
                "value": "none"
            },
            {
                "name": "-webkit-column-rule-width",
                "value": "0px"
            },
            {
                "name": "-webkit-column-span",
                "value": "none"
            },
            {
                "name": "-webkit-column-width",
                "value": "auto"
            },
            {
                "name": "-webkit-filter",
                "value": "none"
            },
            {
                "name": "align-content",
                "value": "start"
            },
            {
                "name": "align-items",
                "value": "start"
            },
            {
                "name": "align-self",
                "value": "start"
            },
            {
                "name": "flex-basis",
                "value": "auto"
            },
            {
                "name": "flex-grow",
                "value": "0"
            },
            {
                "name": "flex-shrink",
                "value": "1"
            },
            {
                "name": "flex-direction",
                "value": "row"
            },
            {
                "name": "flex-wrap",
                "value": "nowrap"
            },
            {
                "name": "justify-content",
                "value": "start"
            },
            {
                "name": "-webkit-font-smoothing",
                "value": "auto"
            },
            {
                "name": "-webkit-highlight",
                "value": "none"
            },
            {
                "name": "-webkit-hyphenate-character",
                "value": "auto"
            },
            {
                "name": "-webkit-line-box-contain",
                "value": "block inline replaced"
            },
            {
                "name": "-webkit-line-break",
                "value": "auto"
            },
            {
                "name": "-webkit-line-clamp",
                "value": "none"
            },
            {
                "name": "-webkit-locale",
                "value": "auto"
            },
            {
                "name": "-webkit-margin-before-collapse",
                "value": "collapse"
            },
            {
                "name": "-webkit-margin-after-collapse",
                "value": "collapse"
            },
            {
                "name": "-webkit-mask-box-image",
                "value": "none"
            },
            {
                "name": "-webkit-mask-box-image-outset",
                "value": "0px"
            },
            {
                "name": "-webkit-mask-box-image-repeat",
                "value": "stretch"
            },
            {
                "name": "-webkit-mask-box-image-slice",
                "value": "0 fill"
            },
            {
                "name": "-webkit-mask-box-image-source",
                "value": "none"
            },
            {
                "name": "-webkit-mask-box-image-width",
                "value": "auto"
            },
            {
                "name": "-webkit-mask-clip",
                "value": "border-box"
            },
            {
                "name": "-webkit-mask-composite",
                "value": "source-over"
            },
            {
                "name": "-webkit-mask-image",
                "value": "none"
            },
            {
                "name": "-webkit-mask-origin",
                "value": "border-box"
            },
            {
                "name": "-webkit-mask-position",
                "value": "0% 0%"
            },
            {
                "name": "-webkit-mask-repeat",
                "value": "repeat"
            },
            {
                "name": "-webkit-mask-size",
                "value": "auto"
            },
            {
                "name": "order",
                "value": "0"
            },
            {
                "name": "perspective",
                "value": "none"
            },
            {
                "name": "perspective-origin",
                "value": "712px 8.5px"
            },
            {
                "name": "-webkit-print-color-adjust",
                "value": "economy"
            },
            {
                "name": "-webkit-rtl-ordering",
                "value": "logical"
            },
            {
                "name": "shape-outside",
                "value": "none"
            },
            {
                "name": "shape-image-threshold",
                "value": "0"
            },
            {
                "name": "shape-margin",
                "value": "0px"
            },
            {
                "name": "-webkit-tap-highlight-color",
                "value": "rgba(0, 0, 0, 0.4)"
            },
            {
                "name": "-webkit-text-combine",
                "value": "none"
            },
            {
                "name": "-webkit-text-decorations-in-effect",
                "value": "none"
            },
            {
                "name": "-webkit-text-emphasis-color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "-webkit-text-emphasis-position",
                "value": "over"
            },
            {
                "name": "-webkit-text-emphasis-style",
                "value": "none"
            },
            {
                "name": "-webkit-text-fill-color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "-webkit-text-orientation",
                "value": "vertical-right"
            },
            {
                "name": "-webkit-text-security",
                "value": "none"
            },
            {
                "name": "-webkit-text-stroke-color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "-webkit-text-stroke-width",
                "value": "0px"
            },
            {
                "name": "transform",
                "value": "none"
            },
            {
                "name": "transform-origin",
                "value": "712px 8.5px"
            },
            {
                "name": "transform-style",
                "value": "flat"
            },
            {
                "name": "-webkit-user-drag",
                "value": "auto"
            },
            {
                "name": "-webkit-user-modify",
                "value": "read-only"
            },
            {
                "name": "-webkit-user-select",
                "value": "text"
            },
            {
                "name": "-webkit-writing-mode",
                "value": "horizontal-tb"
            },
            {
                "name": "-webkit-app-region",
                "value": "no-drag"
            },
            {
                "name": "buffered-rendering",
                "value": "auto"
            },
            {
                "name": "clip-path",
                "value": "none"
            },
            {
                "name": "clip-rule",
                "value": "nonzero"
            },
            {
                "name": "mask",
                "value": "none"
            },
            {
                "name": "filter",
                "value": "none"
            },
            {
                "name": "flood-color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "flood-opacity",
                "value": "1"
            },
            {
                "name": "lighting-color",
                "value": "rgb(255, 255, 255)"
            },
            {
                "name": "stop-color",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "stop-opacity",
                "value": "1"
            },
            {
                "name": "color-interpolation",
                "value": "sRGB"
            },
            {
                "name": "color-interpolation-filters",
                "value": "linearRGB"
            },
            {
                "name": "color-rendering",
                "value": "auto"
            },
            {
                "name": "fill",
                "value": "rgb(0, 0, 0)"
            },
            {
                "name": "fill-opacity",
                "value": "1"
            },
            {
                "name": "fill-rule",
                "value": "nonzero"
            },
            {
                "name": "marker-end",
                "value": "none"
            },
            {
                "name": "marker-mid",
                "value": "none"
            },
            {
                "name": "marker-start",
                "value": "none"
            },
            {
                "name": "mask-type",
                "value": "luminance"
            },
            {
                "name": "shape-rendering",
                "value": "auto"
            },
            {
                "name": "stroke",
                "value": "none"
            },
            {
                "name": "stroke-dasharray",
                "value": "none"
            },
            {
                "name": "stroke-dashoffset",
                "value": "0px"
            },
            {
                "name": "stroke-linecap",
                "value": "butt"
            },
            {
                "name": "stroke-linejoin",
                "value": "miter"
            },
            {
                "name": "stroke-miterlimit",
                "value": "4"
            },
            {
                "name": "stroke-opacity",
                "value": "1"
            },
            {
                "name": "stroke-width",
                "value": "1px"
            },
            {
                "name": "alignment-baseline",
                "value": "auto"
            },
            {
                "name": "baseline-shift",
                "value": "0px"
            },
            {
                "name": "dominant-baseline",
                "value": "auto"
            },
            {
                "name": "text-anchor",
                "value": "start"
            },
            {
                "name": "writing-mode",
                "value": "lr-tb"
            },
            {
                "name": "glyph-orientation-horizontal",
                "value": "0deg"
            },
            {
                "name": "glyph-orientation-vertical",
                "value": "auto"
            },
            {
                "name": "vector-effect",
                "value": "none"
            },
            {
                "name": "paint-order",
                "value": "fill stroke markers"
            },
            {
                "name": "cx",
                "value": "0px"
            },
            {
                "name": "cy",
                "value": "0px"
            },
            {
                "name": "x",
                "value": "0px"
            },
            {
                "name": "y",
                "value": "0px"
            },
            {
                "name": "r",
                "value": "0px"
            },
            {
                "name": "rx",
                "value": "0px"
            },
            {
                "name": "ry",
                "value": "0px"
            }
        ]
    }
}
